app/models/katello/presenters/sync_status.rb

```diff
@@ -0,0 +1,69 @@
+module Katello
```

Comment:
Should this live here -- https://github.com/dustints/katello/tree/sync_stat_task/app/presenters/katello